### PR TITLE
Added persistence.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich ,

Adding persistence that we need for the launch. Plays well with https://github.com/c5t/Current/pull/222.

Thanks!
Dima

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,8 @@
 *.json
 *.swp
 *.log
+
+core
+
+./ctfo_events.log
+./db.json
```
Comment (mzhurovich) [inline .gitignore]: Hm, there are `*.log` and `*.json` above.

Comment (dkorolev) [inline .gitignore]: :+1: b925f27

**Comment by mzhurovich** [comment]
@dkorolev I just thought that constructor reading from cards startup file will reset the state read by Yoda.

**Comment by dkorolev** [comment]
Let's chat about it by voice in some 25 .. 45 minutes?

File: .gitignore (hunk 1 above)
Comment (mzhurovich) [inline .gitignore]: Hm, I don't see this commit in the list of this pull request

Comment (dkorolev) [inline .gitignore]: My bad. And now?

**Comment by mzhurovich** [comment]
@dkorolev :+1:

**Pull request merged**